src/Tasks/NativeMethods.cs

```diff
@@ -1129,46 +1127,38 @@ internal static bool AllDrivesMapped()
                     fUnified1,
                     assemblyIdentity2,
                     fUnified2,
-                    out pfEquivalent,
-                    out pResult);
+                    out bool pfEquivalent,
+                    out _);
+                return pfEquivalent;
```

Comment:
Without the new return statement the results of `CompareAssemblyIdentityWindows` were ignored so it looks like it would be safer to not call this method at all and keep comparing identities in C# on all platforms. Is performance the reason why you chose to keep the Windows-only DllImport?

Reply:
On the one hand, going back to the original, long-proven code is nice. On the other, evidently this managed implementation is ok since the return was ignored! I wish we had more testing in this area to keep that true, so I filed #5974. But that doesn't need to block this!

Since this is a very perf-sensitive area, it might be nice to do some testing--if the native implementation is noticeably faster, we should probably keep it. Unfortunately I don't think it'll be easy to isolate that with a good set of test cases.

Reply:
I did some benchmarking and the fusion implementation seems to be ~10 times faster than its managed replacement. I, however, not sure it should mean so much for a decision, as I am not sure we run this code often enough for the difference to be noticeable: it is called only when we have a conflict. It is also clear that the managed implementation is meant to be just a replacement for the absent fusion function (see the TODO comment, line 1114) and I believe it might be less safe than the original fusion.dll code. From the other point of view, this managed code was shipped and also worked so far. So, well, it is hard to decide what is actually safer: to keep the fusion.dll or this managed implementation.

Reply:
Thank you for following up! It would probably make sense to copy your comment to the PR description and consequently to the commit message.

Reply:
Team triage: talked with @ladipro and @Forgind. We now think we should just delete the native call entirely--that's what we've been doing, it's a perf improvement on Windows (since the native call won't happen), and it gets us to a consistent implementation everywhere.